docs/website/docs/tutorials/deploying-a-devfile-using-odo-on-IBM-Z-and-Power.md

```diff
@@ -12,24 +12,26 @@ The images can be used for devfiles on IBM Z & Power
-| Vert.x Java| java-vertx | Upstream Vert.x using Java | registry.redhat.io/codeready-workspaces/plugin-java11-openj9-rhel8 | s390x, ppc64le|
-| Java | java-wildfly-bootable-jar | Java stack with WildFly in bootable Jar mode, OpenJDK 11 and Maven 3.5 | registry.access.redhat.com/ubi8/openjdk-11 | s390x |
-| Node.JS | nodejs | Stack with NodeJS 12 | registry.redhat.io/codeready-workspaces/plugin-java8-openj9-rhel8 | s390x, ppc64le|
-| Node.JS | nodejs-angular | Stack with Angular 12 | node:lts-slim | s390x |
-| Node.JS | nodejs-nextjs | Stack with Next.js 11 | node:lts-slim | s390x |
-| Node.JS | nodejs-nuxtjs | Stack with Nuxt.js 2 | node:lts | s390x |
-| Node.JS | nodejs-react | Stack with React 17 | node:lts-slim | s390x |
-| Node.JS | nodejs-svelte | Stack with Svelte 3 | node:lts-slim | s390x |
-| Node.JS | nodejs-vue | Stack with Vue 3 | node:lts-slim | s390x |
-| PHP | php-laravel | Stack with Laravel 8 | composer:latest | s390x |
-| Python| python | Python Stack with Python 3.7 | registry.redhat.io/codeready-workspaces/plugin-java8-openj9-rhel8 | s390x, ppc64le|
-| Django| python-django| Python3.7 with Django| registry.redhat.io/codeready-workspaces/plugin-java8-openj9-rhel8| s390x, ppc64le|
+| go   | go | Stack with the latest Go version | golang:latest | s390x |
+| java      | java-maven    | Upstream Maven and OpenJDK 11 | rregistry.redhat.io/codeready-workspaces/plugin-java11-rhel8 | s390x, ppc64le |
+| java      | java-openliberty | Java application Maven-built stack using the Open Liberty runtime | icr.io/appcafe/open-liberty-devfile-stack:22.0.0.1 | s390x, ppc64le |
+| java      | java-openliberty-gradle | Java application Gradle-built stack using the Open Liberty runtime | icr.io/appcafe/open-liberty-devfile-stack:22.0.0.1-gradle | s390x |
+| java | java-quarkus | Quarkus with Java | registry.access.redhat.com/ubi8/openjdk-11 | s390x, ppc64le|
+| java | java-springboot | Spring Boot® using Java| registry.redhat.io/codeready-workspaces/plugin-java11-rhel8 | s390x, ppc64le|
+| java | java-vertx | Upstream Vert.x using Java | registry.redhat.io/codeready-workspaces/plugin-java11-rhel8 | s390x, ppc64le|
+| java | java-websphereliberty | Java application Maven-built stack using the WebSphere Liberty runtime | icr.io/appcafe/websphere-liberty-devfile-stack:22.0.0.1 | s390x |
+| java | java-websphereliberty-gradle | Java application Gradle-built stack using the WebSphere Liberty runtime | icr.io/appcafe/websphere-liberty-devfile-stack:22.0.0.1-gradle | s390x |
+| java | java-wildfly-bootable-jar | Java stack with WildFly in bootable Jar mode, OpenJDK 11 and Maven 3.5 | registry.access.redhat.com/ubi8/openjdk-11 | s390x |
+| javascript | nodejs | Stack with Node.js 14 | registry.access.redhat.com/ubi8/nodejs-14:latest | s390x, ppc64le|
+| typescript | nodejs-angular | Stack with Angular 12 | node:lts-slim | s390x |
```

Comment:
Are you sure it's `typescript` and not `javascript`? I personally am not sure, but I think it's `javascript`.

Reply:
I am not sure, just see it from this [devfile](https://github.com/devfile/registry/blob/main/stacks/nodejs-angular/devfile.yaml)... maybe javascript is better to user.

Reply:
OK. It's best to stick with it in that case. I'll check with the folks maintaining the stack to know if it's typescript or javascript. 😄